lua/autorun/client/cfc_attention_monitoring_cl_init.lua

```diff
@@ -1,4 +1,6 @@
 local HasFocus = system.HasFocus
+local eyePos = EyePos
+local eyeAngles = EyeAngles
```

Comment:
nit: You can localize these with the same casing - that's standard practice in our org anyway.

Reply:
ah okay, not that familiar yet